spec/presenters/webauthn_setup_presenter_spec.rb

```diff
@@ -25,7 +25,13 @@
   describe '#image_path' do
     subject { presenter.image_path }
 
-    it { is_expected.to  eq('security-key.svg') }
+    it { is_expected.to eq('security-key.svg') }
+  end
+
+  describe '#page_title' do
+    subject { presenter.image_path }
```

Comment:
Is probably why the build is failing, but I assume this should be using the `page_title` member and not `image_path`. Same note below.

```suggestion
    subject { presenter.page_title }
```